pkgs/applications/science/math/programmer-calculator/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, gcc, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "programmer-calculator";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "alt-romes";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1vvpbj24ijl9ma0h669n9x0z1im3vqdf8zf2li0xf5h97b14gmv0";
+  };
+
+  buildInputs = [ gcc ncurses ];
```

Comment:
```suggestion
  buildInputs = [ ncurses ];
```
part of the stdenv.

Reply:
This was added to make it build on macOS as per my comment above. What is the correct way to fix this problem?

Reply:
darwin by default uses clang. To force gcc please use gccStdenv.

Reply:
Thanks! Switched to `gccStdenv`.